packages/fxa-settings/src/components/FormPhoneNumber/index.tsx

```diff
@@ -62,13 +63,15 @@ const FormPhoneNumber = ({
     countryCode,
   }: InputPhoneNumberData) => {
     setHasErrors(false);
+    setIsSubmitting(true);
     const formattedPhoneNumber = formatPhoneNumber({
       phoneNumber,
       countryCode,
     });
     const result = await submitPhoneNumber(formattedPhoneNumber);
     if (result !== undefined && result.hasErrors) {
       setHasErrors(true);
+      setIsSubmitting(false);
```

Comment:
I guess it doesn't really matter that we are setting that state to false in here, since a successful submission would navigate the user elsewhere(?).  But the is submitting state should be false regardless of the error state of the submission result.